frame/balances/src/benchmarking.rs

```diff
@@ -40,19 +42,18 @@ mod benchmarks {
 	// * Transfer will create the recipient account.
 	#[benchmark]
 	fn transfer_allow_death() {
-		let existential_deposit = T::ExistentialDeposit::get();
+		let existential_deposit: T::Balance = MINIMUM_BALANCE.into();
 		let caller = whitelisted_caller();
 
 		// Give some multiple of the existential deposit
-		let balance = existential_deposit.saturating_mul(ED_MULTIPLIER.into());
```

Comment:
why removing the multiplier? We can keep it

Reply:
For this benchmark we need the account to be reaped so we can no longer transfer a part of the balance, rather the entire balance needs to be transferred so the benchmark case is respected, which doesn't really need the multiplier.

Reply:
I can alternatively also make it to transfer the balance instead of the ED, if that's better.

Reply:
Even if the multiplier is useless, my point is to minimize the changes, change only what we strictly need to be able to run the benchmarks

Reply:
Done.